Utilities/convertFromDepsCloud.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs')
+      path = require('path')
+      url = require('url')
+
+const depsInputPath =
+    process.env.MARINER_DEPSCLOUD_INPUT ||
+    path.join(__dirname, '..', 'InputFiles', 'candidate.json');
+const depsOutputPath =
+    process.env.MARINER_DEPSCLOUD_OUTPUT ||
+    path.join(__dirname, '..', 'OutputFiles', 'candidate-converted.json');
+
+function convert(deps) {
+  result = {};
+  for(var entry in deps) {
+    const fullPath = url.parse(deps[entry].repository_url).pathname;
+    var repoScore = deps[entry].score;
+    // Assuming all URIs would have a double slash, we follow the slashes to get owner and repo
+    var owner = fullPath.split("/")[1];
+    var repo = fullPath.split("/")[2];
```

Comment:
Instead of parsing the string twice, I'd parse it once, validate the length is what you expect it to be (>=2), and then pull out the values.

Reply:
This would work in isolation. We will run into situations where the path includes more than the owner and repo, and this is where I find the split useful.

I have adjusted this section to only include one split, which is a great idea:
```
repoPath = fullPath.split("/");
var owner = repoPath[1];
var repo = repoPath[2];
```